fetch_moveit_config/launch/sensor_manager.launch.xml

```diff
@@ -1,7 +1,12 @@
 <launch>
 
+  <!-- Set the file describing which sensor(s) to use to build the octomap  -->
+  <arg name="moveit_octomap_sensor_params_file" default="$(find fetch_moveit_config)/config/sensors.yaml" />
```

Comment:
Why is this default not applied to the file `move_group.launch`

Reply:
I just added the default.